packages/editor-ui/src/modules/nodeTypes.ts

```diff
@@ -85,3 +134,13 @@ function toNodeTypesState(nodeTypes: INodeTypeDescription[]): INodeTypesState['n
 }
 
 export default module;
+
+function hasValidVersion(nodeType: INodeTypeDescription, version?: number) {
+	const nodeTypeVersion = Array.isArray(nodeType.version)
+		? nodeType.version
+		: [nodeType.version];
+
+	return nodeTypeVersion.includes(version || nodeType.defaultVersion || DEFAULT_NODETYPE_VERSION);
+}
+
+const omit = (keyToOmit: string, { [keyToOmit]: _, ...remainder }) => remainder;
```

Comment:
love that you did not use lodaash.. maybe we should move this into a helper file

Reply:
Done!